docs/registry/configure.md

```diff
@@ -206,26 +159,34 @@ The custom model is configured as a component with the name "my-realsense".
 {{% /tab %}}
 {{% /tabs %}}
 
-### Edit the module
+{{% /tab %}}
+{{< /tabs >}}
+
+{{< alert title="Note" color="note" >}}
+If you are using a [rented Viam rover](/appendix/try-viam/), adding modules is disabled for security purposes.
+{{< /alert >}}
 
-You can also view and edit the configuration of a modular resource's underlying module from the **CONFIGURE** tab:
+{{< /expand >}}
 
-1.  Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
-2.  Scroll to the card with the name of the module, as indicated by its page in the [registry](https://app.viam.com/registry).
+### Further configuration options
 
-    This is the **Registry module** card, which lists the deployed version of the module and the latest version available.
-    Here you can [configure how a module updates](#configure-version-update-management-for-a-registry-module).
+{{< expand "Add additional modular resources from the same module" >}}
 
-    {{<imgproc src="registry/configure/deployed-module-with-component.png" style="width: 400px" resize="500x" declaredimensions=true alt="The module card">}}
+Once you have [added a module](#configure-a-modular-resource-on-your-machine) from the Viam registry, you can add any number of the modular resources it provides to your machine by adding new components or services configured with your modular resource's {{< glossary_tooltip term_id="model" text="model" >}}.
 
-    You can also use JSON mode to [configure environment variables](#use-environment-variables-with-a-registry-module) for your module.
+Follow the same steps as when you added the first modular resource, clicking **Create** and **Component** or **Service** as applicable.
+You will be prompted to click **Add module** again while configuring the resource, though no duplicate module will be added to the `modules` section of the configuration.
 
-### Configure version update management for a registry module
+{{< /expand >}}
+{{< expand "Configure version update management" >}}
```

Comment:
Ah interesting - you can merge this with the config table I suggest above. Maybe put an expander with more info underneath the table. All this is a bit much to put in the attribute description but it is helpful to spell out further probably.